### DISCUSSION TITLE
Vehicles state is always "offline" and not "asleep" like before

### DISCUSSION THREAD

**Discussion by natrlhy** [teslamate-org/teslamate#4067] [Q&A] [answered]
For the past few months i'd say, the vehicle's state when it's idle (haven't opened the app and the car is supposed to be asleep, etc), teslamate reports that it's just "offline". Is this the new default state teslamate reports vehicles in now or is something wrong?

Everything else seems to work and get logged but I wanted to see how I can dig into why it's always "offline" now.

**Comment by cwanja** [comment, marked as answer]
Please see #3753. Healthy discussion there. And our reason for not changing.

**Reply by natrlhy** [reply]
Thanks @cwanja! Excellent details there. Sorry I didn't see it before
